package.json

```diff
@@ -9,7 +9,7 @@
     "grunt-contrib-concat": "~0.4.0",
     "grunt-contrib-copy": "~0.5.0",
     "grunt-contrib-cssmin": "~0.9.0",
-    "grunt-contrib-jshint": "~0.6.0",
+    "grunt-eslint": "^7.0.1",
```

Comment:
Do we want to be this broad? I don't know about eslint, but I've found auto-updating Node packages on minor versions to introduce breaks.